core/src/main/java/com/yahoo/tagchowder/Parser.java

```diff
@@ -1185,6 +1195,9 @@ private String makeName(final char[] buff, final int offset, final int length) {
                 }
                 start = true;
                 dst.append(translateColons ? '_' : ch);
+            } else if (ampValidation && (ch == '[' || ch == ']' || ch == '{' || ch == '}' || ch == '⚡')) {
```

Comment:
Shouldn't we allow [, { and ⚡ only if "start=true"?

Reply:
], } can be allowed if its the last char in the attribute name

Reply:
Good point! fixed